cli/users.go

```diff
@@ -104,27 +104,48 @@ func (hbc *HoneybadgerClient) DeleteUser(userID int, teamID int) error {
 	return nil
 }
 
-// GetUserFromTeams - Get User information from Teams
+// GetUserFromTeams - Get Users information from Teams
 func (hbc *HoneybadgerClient) GetUserFromTeams(userEmail string) (userTeams []HoneybadgerUser, err error) {
+	insertedUser := make(map[string]bool)
+
 	teams, err := hbc.GetTeams()
 	if err != nil {
 		return userTeams, err
 	}
 
 	for _, team := range teams {
+		delete(insertedUser, userEmail) // Ensure that It is only valid fro the current team
```

Comment:
If this is the case, why not just creating a boolean here?
```
insertedUser := false
```
then, substitute `insertedUser[userEmail] = true` by `insertedUser = true` and finally in the check, only  check by the value of this boolean?

Sorry, but I don't see the benefit of using a map here. Could you explain me which are those?
and then in the conditional check for the

Reply:
Good point 😄